Headers/AppKit/NSEvent.h

```diff
@@ -36,6 +36,19 @@
 // For NSTimeInterval
 #import <Foundation/NSDate.h>
 
+#if OS_API_VERSION(MAC_OS_X_VERSION_10_7, GS_API_LATEST)
```

Comment:
Sorry, I did commit just this change earlier today. I was from your mail under the impression that you expected me to implement this.